ds3-cli-certification/src/test/java/com/spectralogic/ds3cli/certification/Certification_Test_8.java

```diff
@@ -0,0 +1,403 @@
+            OUT.finishTest(testDescription, success);
+            assertTrue(success);
+        }
+    }
+
+    /**
+     * 8.6: Show a job is fully persisted to tape or disk archive location
+     */
+    @Test
+    public void test_8_6_fully_persisted() throws Exception {
+        final String testDescription = "8.6: Show Fully Persisted";
+        final Integer numFiles = 6;
+        final Long fileSize = GB_BYTES;
+        final String bucketName = CertificationUtil.getBucketName(testDescription);
+        boolean success = false;
+
+        OUT.startNewTest(testDescription);
+        try {
+            // Assume there is a valid Tape Partition
+            final UUID tapePartitionId = getValidTapePartition(client);
+            assumeThat(tapePartitionId, is(notNullValue()));
+
+            // Un-quiesce the tape partition to allow cache offload to tape
+            if (!ensureTapePartitionQuiescedState(client, tapePartitionId, Quiesced.NO)) {
+                final String tapePartitionStateChangeErrorMsg = "Timed out waiting for TapePartition " + tapePartitionId + " Quiesced State to change to Quiesced.NO";
+                OUT.insertLog(tapePartitionStateChangeErrorMsg);
+                fail(tapePartitionStateChangeErrorMsg);
+            }
+
+            // Put files into bucket
+            // Create temp files for BULK_PUT
+            final Path bulkPutLocalTempDir = CertificationUtil.createTempFiles(bucketName, numFiles, fileSize);
+
+            final String putBulkCmd = "--http -c put_bulk -b " + bucketName + " -d "  + bulkPutLocalTempDir.toString();
+            final CommandResponse putBulkResponse = Util.command(client, putBulkCmd);
+            assertThat(putBulkResponse.getReturnCode(), is(0));
+
+            // verify write
+            OUT.insertLog("List bucket " + bucketName + "contents");
+            final Iterator<Contents> objects = HELPERS.listObjects(bucketName).iterator();
+            assertTrue(objects.hasNext());
+            final String objectName = objects.next().getKey();
+            OUT.insertPreformat(objectName);
+
+            // Show not persisted
+            OUT.insertLog("Show not persisted");
+            final String getPhysicalPlacementCmd = "--http -c get_detailed_objects_physical -b " + bucketName;
+            final CommandResponse getPhysicalPlacementBeforeResponse = Util.command(client, getPhysicalPlacementCmd);
+            OUT.insertCommand(getPhysicalPlacementCmd, getPhysicalPlacementBeforeResponse.getMessage());
+
+            // force to tape
+            OUT.insertLog("Reclaim cache (force to tape)");
+            final String reclaimCacheArgs = "--http -c reclaim_cache";
+            final CommandResponse reclaimResponse = Util.command(client, reclaimCacheArgs);
+            OUT.insertCommand(reclaimCacheArgs, reclaimResponse.getMessage());
+            assertThat(reclaimResponse.getReturnCode(), is(0));
+
+            // Show persisted
+            final int MAX_POLL_RETRIES = 50;
```

Comment:
There is similar functionality in some of the 7.x tests.  Please attempt to consolidate this logic with that if possible.